src/modules/browse/admin.ts

```diff
@@ -0,0 +1,24 @@
+import * as opn from 'opn'
+import * as conf from '../../conf'
+
+const buildPrefix = (account: string, workspace: string): string => {
+  const base = 'https://'
+  const workspacePrefix = workspace === 'master' ? '' : `${workspace}--`
```

Comment:
Since you are sending the user to their current workspace, you should always set the prefix. Keep in mind that no prefix is not the same as master, it means any workspace that is set to receive production traffic and has positive weight. If you want to use the current workspace set in the context, you should make `master` explicit as well.

From my understanding, this will probably make this function a simple one-liner that you can put directly in the place that is using it. 🙂

Reply:
Well, that's neat. Fixed